jaq/src/cli.rs

```diff
@@ -121,6 +122,14 @@ impl Cli {
             "version" => self.short('V', args)?,
             "help" => self.short('h', args)?,
 
+            "img" => {
+                let value = args.next().ok_or(Error::Flag("--img".to_string()))?;
+                if value == "auto" {
+                    self.img_auto = true;
+                } else {
+                    return Err(Error::Flag(format!("--img {}", value.to_string_lossy())));
```

Comment:
Do you envision something like `--img png`, `--img jpeg` here in order to restrict image formats?

Reply:
Currently it's using image library that detects image format and we are guessing base64 encoding,
however I envision that in future people may come with other types of concodings or more image formats or whatever, so starting with "auto" gives a nice room for adding support for more staff if needed... I was just thinking about more future proof api to not break in future backwards CLI interface compatibility, but ofc if you think otherwise, we can change it.

The other think that could be considered or this used for would be to provide differnt ways or resizing or firmatting like "--img {auto, scale:2.0}" or "--img {auto, no_identation}" ...

so making it now "auto" leaves room open. That was my intent.